### PR TITLE
enhancement: Other queries for shared snapshots

### PR DESCRIPTION
**Pull request by michelvocks**
Closes https://github.com/cloudquery-policies/aws/issues/106

I wasn't sure if it makes more sense to add a new "other" folder below every sub-api-endpoint instead of the top level. Happy to change that if necessary.

### CODE REVIEW

**Review by yevgenypats — CHANGES_REQUESTED** [review]
I think the right structure here should be all the `queries` and `queries` and have `other.hcl` and add it to https://github.com/cloudquery-policies/aws/blob/main/policy.hcl . This way we keep the same convention + it will get indexed in hub.cloudquery.io

**Comment by yevgenypats** [comment]
Another point: without it being in a subpolicy a user can't run it in the command line via `policy` command

**Review by yevgenypats — APPROVED** [review]
LGTM. just need to fix the title for the CI to pass

**Comment by roneli** [comment]
would add the query in #107 and add user as contributor to this PR.

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,33 +0,0 @@
...
-  push:
-    branches:
-      - main
-  pull_request:
-
-jobs:
-  LintHCL:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Set up Go 1.x
```
Comment (yevgenypats) [inline .github/workflows/lint.yml:13]: maybe keep the hcl formatter ?

**Review by yevgenypats — CHANGES_REQUESTED** [review]
Had one comment on the linter. also looks like the second test it failing for some reason.

**Comment by erezrokah** [comment]
Going to close this as stale. If I got this wrong please let me know and I'll re-open

**Pull request closed without merging**